dydx3/modules/private.py

```diff
@@ -1185,6 +1185,62 @@ def get_profile(
         '''
         return self._get('profile/private', {})
 
+    def get_user_links(
+        self,
+    ):
+        '''
+        Get Active Linked Users
+
+        :returns: UserLinks
```

Comment:
Where's `UserLinks` defined?

Reply:
we typically don't define return objects in the python client, they have to look at API documentation